etna/transforms/embeddings/models/base.py

```diff
@@ -0,0 +1,42 @@
+from abc import abstractmethod
+
+import numpy as np
+import pandas as pd
+
+from etna.core import BaseMixin
+from etna.core import SaveMixin
+
+
+class BaseEmbeddingModel(BaseMixin, SaveMixin):
```

Comment:
Where load and save methods? Why aren't they here?

Reply:
i forgot that in utils base class you didn't implement such methods
i think because in model class there is super of load and save and it did nothing